bulk-patch.js

```diff
@@ -0,0 +1,95 @@
+const axios = require("axios");
+const parseLinkHeader = require('parse-link-header');
```

Comment:
You are mixing single and double quotes. `yarn prettier --write bulk-patch.js ` should fix all such issues.